src/containers/App.jsx

```diff
@@ -102,12 +102,9 @@ const App = ({
   }, [hasError, isFetching, status])
 
   useEffect(() => {
-    if (isReady) {
-      if (isFlagshipApp() && webviewIntent) {
-        webviewIntent.call('hideSplashScreen')
-      }
-    }
-  }, [webviewIntent, isReady])
+    isReady && appsReady && webviewIntent?.call('hideSplashScreen')
```

Comment:
if we really need this new appsReady state, then add it to `isReady`. 

```js
if(isReady) hideSplashScren()


const isReady = appsReady && .. && .. 
```

Reply:
✅